REQUIRE

```diff
@@ -1 +1 @@
-julia 0.5-
+julia 0.6
```

Comment:
use 0.6.0-pre or one of the rc's as a lower bound if you want it to be installable now